Sources/PackageLoading/ManifestLoader.swift

```diff
@@ -17,9 +17,11 @@ import TSCBasic
 import enum TSCUtility.Diagnostics
 
 public enum ManifestParseError: Swift.Error, Equatable {
+    /// The manifest is empty, or at least from SwiftPM's perspective it is.
+    case emptyManifest(path: AbsolutePath)
     /// The manifest contains invalid format.
     case invalidManifestFormat(String, diagnosticFile: AbsolutePath?)
-
+    // TODO: Test this error.
```

Comment:
this TODO is beyond this PR, correct? iow this PR is ready to go?

Reply:
Yep and yep :)